packages/tasit-account/src/ethers.test.js

```diff
@@ -0,0 +1,176 @@
+import { expect, assert } from "chai";
+import { ethers } from "ethers";
+
+const waitForEvent = async (eventName, expected) => {
+	return new Promise(function(resolve, reject) {
+		contract.on(eventName, function() {
+			const args = Array.prototype.slice.call(arguments);
+			const event = args.pop();
+			event.removeListener();
+			expect(args, `${event.event} event should have expected args`).to.deep.equal(
+				expected
+			);
+			resolve();
+		});
+	});
+};
+
+const ropstenProvider = ethers.getDefaultProvider("ropsten");
+const zero = ethers.utils.bigNumberify(0);
+
+let wallet, randomWallet, contract;
+
+const rawTx = {
+	nonce: 0,
+	gasLimit: 21000,
+	gasPrice: ethers.utils.bigNumberify("20000000000"),
+	to: "0x88a5C2d9919e46F883EB62F7b8Dd9d0CC45bc290",
+	value: ethers.utils.parseEther("0.0000001"),
+	data: "0x",
+	chainId: ethers.utils.getNetwork("ropsten").chainId,
+};
+
+describe("ethers.js - pretests", function() {
```

Comment:
@marcelomorgado Oh, I suppose I should have added the pretests comment in this PR since this got moved. Did you use `git mv` to rename the file? If so, I think the change might have made it through to here.

Reply:
Maybe once we resolve the merge conflicts this will be working, though

Reply:
Oooh, once you merged develop into this branch it did work!